clients/da-vinci-client/src/main/java/com/linkedin/davinci/client/AvroGenericDaVinciClient.java

```diff
@@ -369,16 +375,21 @@ public CompletableFuture<Map<K, V>> batchGet(Set<K> keys) {
   }
 
   @Override
-  public ComputeRequestBuilder<K> compute() throws VeniceClientException {
-    return compute(Optional.empty(), Optional.empty(), 0);
+  public void streamingBatchGet(Set<K> keys, StreamingCallback<K, V> callback) throws VeniceClientException {
```

Comment:
Yeah, I had added it, but I couldn't get the test coverage to pass. Since it was not required right now, I added the unsupported operation exception. Let me give it one more try

Reply:
Yeah, it's running into coverage issues. I can add this in a future PR. Does that work?

This will be the implementation:
```
  @Override
  public void streamingBatchGet(Set<K> keys, StreamingCallback<K, V> callback) throws VeniceClientException {
    throw new VeniceUnsupportedOperationException("streamingBatchGet for DaVinci client");
    // DaVinci client doesn't do streaming batch get, so implement streaming batch get using the batch get functionality
    CompletableFuture<Map<K, V>> batchGetResponseFuture = batchGetImplementation(keys);
    batchGetResponseFuture.whenComplete((batchGetResponse, throwable) -> {
      if (throwable != null) {
        // The callback can be completed exceptionally
        callback.onCompletion(Optional.of(new VeniceClientException("Request failed with exception ", throwable)));
        return;
      }

      for (Map.Entry<K, V> responseEntry: batchGetResponse.entrySet()) {
        callback.onRecordReceived(responseEntry.getKey(), responseEntry.getValue());
      }

      Set<K> missingKeys = new HashSet<>(keys);
      missingKeys.removeAll(batchGetResponse.keySet());

      for (K missingKey: missingKeys) {
        callback.onRecordReceived(missingKey, null);
      }

      callback.onCompletion(Optional.empty());
    });
  }
```

Reply:
That will be fine.